runtime/gear/src/lib.rs

```diff
@@ -16,6 +16,7 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
+#![feature(const_option)]
```

Comment:
Can we use stable rust to compile after that feature enabled ?

Reply:
Unfortunately we cannot then use this solution. May be  better to change `saturating_div` to common `div`. Do you know the difference between them. Actually I think they have no difference))

Reply:
Removed with @shamilsan advice